src/commands/init.ts

```diff
@@ -70,6 +70,11 @@ const choices = [
     name: "Extensions: Set up an empty Extensions manifest",
     checked: false,
   },
+  {
```

Comment:
this should be behind a experiment flag - we don't want users to see this as a choice.

Checkout https://github.com/firebase/firebase-tools/pull/5465 to see an example of how you can create a new experimental feature and to use it to change behavior of init!